### PR TITLE
DDP-6033 New PDF Angular documents, update Facebook link

### PR DESCRIPTION
**Pull request by MocanaAtBroad**
…n of page

### CODE REVIEW

File: ddp-workspace/projects/ddp-mbc/src/app/components/data-release/data-release.component.ts [hunk 1]
```diff
@@ -167,7 +167,7 @@ import { DisclaimerComponent, ToolkitConfigurationService } from 'toolkit';
                                         <span translate>Toolkit.DataRelease.Appendices.AppendicesList.Item2.Text</span>
                                     </li>
                                     <li class="PageContent-text PageContent-text-list">
-                                        <a href="ConsentAndRelease.pdf" target="_blank" class="Link" translate>Toolkit.DataRelease.Appendices.AppendicesList.Item3.Link</a>
+                                        <a [href]="'Toolkit.DataRelease.ConsentAndReleaseDocument' | translate" target="_blank" class="Link">{{'Toolkit.DataRelease.Appendices.AppendicesList.Item3.Link' | translate}}</a>
```
Comment (MocanaAtBroad) [inline ddp-workspace/projects/ddp-mbc/src/app/components/data-release/data-release.component.ts:170]: Whenever we have a Spanish version of the document, I stuck the document name into the json translation files.

File: ddp-workspace/projects/ddp-mbc/src/app/components/more-details/more-details.component.ts [hunk 2]
```diff
@@ -139,19 +142,20 @@ export class MoreDetailsComponent implements OnInit {
...
+                private ngxTranslate: NGXTranslateService) { }
 
     public ngOnInit(): void {
         this.infoEmail = this.toolkitConfiguration.infoEmail;
         this.infoEmailHref = `mailto:${this.toolkitConfiguration.infoEmail}`;
         this.phone = this.toolkitConfiguration.phone;
         this.phoneHref = `tel:${this.toolkitConfiguration.phone}`;
         this.countMeInUrl = this.toolkitConfiguration.countMeInUrl;
-        this.facebookUrl = `https://www.facebook.com/${this.toolkitConfiguration.facebookGroupId}`;
+        this.facebookUrl$ = this.ngxTranslate.getTranslation('Toolkit.Footer.FacebookLink');
```
Comment (MocanaAtBroad) [inline ddp-workspace/projects/ddp-mbc/src/app/components/more-details/more-details.component.ts:158]: We were getting the Facebook link from the configuration, but now that single value is not going to work for us. so getting it from translation files.

File: ddp-workspace/projects/toolkit/src/lib/components/footer/footer.component.ts [hunk 3]
```diff
@@ -93,17 +95,23 @@ export class FooterComponent implements OnInit {
...
-        this.facebookUrl = `https://www.facebook.com/${this.toolkitConfiguration.facebookGroupId}`;
+        // this.facebookUrl = `https://www.facebook.com/${this.toolkitConfiguration.facebookGroupId}`;
         this.twitterUrl = `https://twitter.com/${this.toolkitConfiguration.twitterAccountId}`;
         this.countMeInUrl = this.toolkitConfiguration.countMeInUrl;
         this.blogUrl = this.toolkitConfiguration.blogUrl;
+        const fbLinkKey = 'Toolkit.Footer.FacebookLink';
+        this.facebookUrl$ = this.ngxTranslate.getTranslation(fbLinkKey).pipe(
+            map(transVal => transVal === fbLinkKey ?
+                `https://www.facebook.com/${this.toolkitConfiguration.facebookGroupId}` : transVal)
+        );
```
Comment (MocanaAtBroad) [inline ddp-workspace/projects/toolkit/src/lib/components/footer/footer.component.ts:113]: The footer component is shared, so tried to make this change backwards compatible. We look for link in translation file and if not found there, we use the toolkit configuration.

File: ddp-workspace/projects/toolkit/src/lib/components/footer/footer.component.ts [hunk 4]
```diff
@@ -93,17 +95,23 @@ export class FooterComponent implements OnInit {
...
+        @Inject('toolkit.toolkitConfig') private toolkitConfiguration: ToolkitConfigurationService,
+        private ngxTranslate: NGXTranslateService) { }
 
     public ngOnInit(): void {
         this.phone = this.toolkitConfiguration.phone;
         this.email = this.toolkitConfiguration.infoEmail;
         this.phoneHref = `tel:${this.toolkitConfiguration.phone}`;
         this.emailHref = `mailto:${this.toolkitConfiguration.infoEmail}`;
-        this.facebookUrl = `https://www.facebook.com/${this.toolkitConfiguration.facebookGroupId}`;
+        // this.facebookUrl = `https://www.facebook.com/${this.toolkitConfiguration.facebookGroupId}`;
```
Comment (aglushko-broad) [inline ddp-workspace/projects/toolkit/src/lib/components/footer/footer.component.ts]: should we remove the commented line?

**Review by aglushko-broad — DISMISSED** [review]

**Review by aglushko-broad — APPROVED** [review]

File: ddp-workspace/projects/toolkit/src/lib/components/footer/footer.component.ts (hunk 4 above)
Comment (MocanaAtBroad) [reply ddp-workspace/projects/toolkit/src/lib/components/footer/footer.component.ts]: Yes! Thnx!

**Pull request merged**